internal/telemetry/sensitivemetadataallowlist/sensitivemetadataallowlist_test.go

```diff
@@ -186,5 +195,38 @@ func TestEventTypesRedact(t *testing.T) {
 			assert.Equal(t, redactAllSensitive, mode)
 			assert.Nil(t, ev.Parameters.PrivateMetadata)
 		})
+		t.Run("allowlisted with wildcard(*) action", func(t *testing.T) {
+			allowedTypes := eventTypes(EventType{
+				Feature:                    "example",
+				Action:                     "*",
+				AllowedPrivateMetadataKeys: []string{"foo"},
+			})
+			ev := &v1.Event{
+				Feature: "example",
+				Action:  "randomAction",
+				Parameters: &v1.EventParameters{
+					PrivateMetadata: &structpb.Struct{
+						Fields: map[string]*structpb.Value{
+							"foo": {
+								Kind: &structpb.Value_StringValue{
+									StringValue: "allowed",
+								},
+							},
+							"bar": {
+								Kind: &structpb.Value_StringValue{
+									StringValue: "redacted",
+								},
+							},
+						},
+					},
+				}}
+			mode := allowedTypes.Redact(ev)
+
+			assert.Equal(t, redactMarketingAndUnallowedPrivateMetadataKeys, mode)
```

Comment:
Sorry for another spacing nit 😆 

```suggestion
			mode := allowedTypes.Redact(ev)
			assert.Equal(t, redactMarketingAndUnallowedPrivateMetadataKeys, mode)
```

Reply:
oh you're good! Do you have a linter you use, I'm surprised its not auto-corrected sometimes

Reply:
No, this is just eyeing it and see if it looks good to read :)